public/js/cita.json

```diff
@@ -5008,5 +4868,305 @@
     "lng": -2.5934541,
     "id": "94e94f50-1d30-49f0-8959-b2a799ba3423",
     "booking_location_id": "b74377da-d06b-48d0-9d74-cc362bbb53b3"
+  },
+  {
+    "title": "Poyton Civic Hall",
```

Comment:
I wasn't sure at all about this place. Google maps [displays it](https://www.google.co.uk/maps/place/Poynton+Civic+Hall/@53.3494361,-2.1196398,17z/data=!3m1!4b1!4m2!3m1!1s0x487a4b160cb25be9:0x7f7b5dfa973c3db8) as Poy**n**ton..